draft-terbu-sd-jwt-vc.md

```diff
@@ -154,34 +158,21 @@ TBD: explain use cases of the three-party-model.
 TBD: conventional crypt, hardware security, hsm, mobile secure area,
 compliance with FIPS
 
-# Verifiable Credentials based on SD-JWT
-
-This section defines encoding, validation and processing rules for SD-JWT VCs.
-
-## Media Type
-
-SD-JWT VCs compliant with this specification MUST use the media type
-`application/vc+sd-jwt` as defined in (#application-vc-sd-jwt).
-
-## Data Format
-
-SD-JWT VCs MUST be encoded using the SD-JWT Combined Format for Issuance as
-defined in Section 5.3. of [@!I-D.ietf-oauth-selective-disclosure-jwt].
+# Common Data Format for JWT VC and SD-JWT VC
 
-SD-JWT VCs MUST contain all Disclosures corresponding to their SD-JWT component
-except for Decoy Digests as per Section 5.1.1.3. of [@!I-D.ietf-oauth-selective-disclosure-jwt].
+JWT VCs and SD-JWT VCs MUST follow the common rules defined in this section.
 
-### Header Parameters
+## Header Parameters
 
-This section defines JWT header parameters for the SD-JWT component of the
-SD-JWT VC.
+This section defines JOSE header parameters for the JWT VC
+and the SD-JWT VC.
 
-The `typ` header parameter of the SD-JWT MUST be present. The `typ` value MUST
-use `vc+sd-jwt`. This indicates that the payload of the SD-JWT contains plain
-JSON and follows the rules as defined in this specification. It further
-indicates that the SD-JWT is a SD-JWT component of a SD-JWT VC.
+The `typ` header parameter MUST be present. The `typ` value MUST
+use the values defined in (#media-type-jwt-vc) or (#media-type-sd-jwt-vc).
```

Comment:
Should we consider to use the same `typ` header value for both variants, JWT and SD-JWT? I came across this when I was thinking about the OID4VC credential format profile for JWT-based VCs. A single `typ` would also result in a single format identifier in the presentation request, which might everyone's live easier. The only reason to distinguish would be that a dedicated format identifier for sd-jwt would allow the processor to directly decide whether selective disclosure is allowed in the presentation definition.

Reply:
I'm fine starting with distinct "typ" media types, as it provides recipients an indication of whether the VC is a plain JWT or SD-JWT.  We could later decide to make that distinction using "cty" if we decide to go in that direction.

Reply:
it's not just the presentation request that needs to differentiate whether there is a selective disclosure or not. how would issuer, verifier, wallets indicate that they support only sd-jwt vc in their metadata? processing of sd-jwt vc and jwt vc are different enough to have separate `typ` values.

@danielfett @bc-pi what do you think?

Reply:
create an issue

Reply:
I can't help but wonder if it'd make sense to have the spec be only about SD-JWT VCs and treat the plain JWT VC as just a special case of an SD-JWT with no selectively disclosable claims.

Reply:
that's what @tlodderstedt has suggested, too. would that change the structure in this PR quite a lot? since the text will be all about sd-jwt vc and there will be one section saying that to do sd-jwt vc, one only need to not use `_sd` and no disclosures?

Reply:
Yeah, I think it would change this PR quite a lot. Or a new one... Sorry!* I'm honestly not 100% sure about the idea but my intuition is that it would be a cleaner/better/simpler approach. 

* I did mention "Isn't a JWT VC just a SD-JWT VC without any selectively disclosable claims... ?" last week in some slack conversation though so this shouldn't be a total surprise.